Cargo.toml

```diff
@@ -86,16 +90,26 @@ swap = [
   "nightly",
   "memmap2",
 ]
+std_fs = []
 
 # Shortcuts
-all_except_failpoints = ["internals", "scripting", "nightly", "swap"]
-all_stable = ["internals", "scripting", "failpoints"]
-all_stable_except_failpoints = ["internals", "scripting"]
+all_unix_stable_without_failpoints = ["internals", "scripting"]
+all_unix_stable = ["all_unix_stable_without_failpoints", "failpoints"]
+
+all_unix_nightly_without_failpoints = ["internals", "scripting", "nightly", "swap"]
+all_unix_nightly = ["all_unix_nightly_without_failpoints", "failpoints"]
+
+all_stable_without_failpoints = ["internals", "scripting", "std_fs"]
+all_stable = ["all_stable_without_failpoints", "failpoints"]
+
+all_nightly_without_failpoints = ["internals", "scripting", "nightly", "swap", "std_fs"]
+all_nightly = ["all_nightly_without_failpoints", "failpoints"]
 
 [patch.crates-io]
 raft-proto = { git = "https://github.com/tikv/raft-rs", branch = "master" }
 protobuf = { git = "https://github.com/pingcap/rust-protobuf", branch = "v2.8" }
 protobuf-codegen = { git = "https://github.com/pingcap/rust-protobuf", branch = "v2.8" }
+grpcio = { git = "https://github.com/tabokie/grpc-rs", branch = "v0.10.x-win" }
```

Comment:
Add a comment to explain the build error. Ref https://github.com/tikv/grpc-rs/pull/622.